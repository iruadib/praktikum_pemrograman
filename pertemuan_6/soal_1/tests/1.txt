3
APL
Apel
40000
1
JRB
Jeruk bali
20000
3
ALP
Alpukat
50000
2