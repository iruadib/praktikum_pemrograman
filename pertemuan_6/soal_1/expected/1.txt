Jumlah barang: 6
Rata-rata harga barang: 33333.33
